code/modules/events/event_container.dm

```diff
@@ -162,6 +162,7 @@ GLOBAL_LIST_EMPTY(event_last_fired)
 		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Meteor Shower",			/datum/event/meteor_wave,				0,		list(ASSIGNMENT_ENGINEER = 25)),
 		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Solar Flare",				/datum/event/solar_flare,				0,		list(ASSIGNMENT_ENGINEER = 25)),
 		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Meaty Ores",				/datum/event/dust/meaty,				0,		list(ASSIGNMENT_ENGINEER = 20)),
+		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Stowaway Changelings",		/datum/event/meteor_wave/gore/ling,		40,		list(ASSIGNMENT_SECURTY  = 20)),
```

Comment:
```suggestion
		new /datum/event_meta(EVENT_LEVEL_MODERATE, "Stowaway Changelings",		/datum/event/meteor_wave/gore/ling,		40,		list(ASSIGNMENT_SECURITY  = 20)),
```
This is currently not scaling. This will need to be changed so it can be observed properly.

Reply:
does that mean the midround doesn't spawn or just doesn't check for sec?

Reply:
It is not properly adjusting the weight of the event in the table, because it doesn't know what to adjust for. So it could spawn, but the chance of it happening naturally right now are low.

Reply:
> but the chance of it happening naturally right now are low.
explains why it hasn't rolled yet on the testmerge hope this gets fixed soon